HW0/P6/P6.txt

```diff
@@ -1 +1,38 @@
-# Your answers here
+Count to Ten
+----------------
+
+Sample Output:
+Hi Job 0
+Hi Job 1
+Hi Job 2
+Hi Job 3
+Bye Job 3
+Bye Job 1
+Bye Job 0
+Bye Job 2
+Hi Job 4
+Hi Job 5
+Hi Job 6
+Hi Job 7
+Bye Job 4
+Hi Job 8
+Bye Job 5
+Hi Job 9
+Bye Job 6
+Bye Job 7
+Bye Job 8
+Bye Job 9
```

Comment:
Writing to the same file is definitely a good example. What I also find occasionally in output:
Hi Job 0
+Hi Job 1
+Hi Job 2
+Hi Job 3
+Bye Job 0Bye Job 2
+
+Bye Job 3Bye Job 1

It seems sometimes two processors output to the single io at the same time. This might cause deadlock due to shared resources. Not 100% sure and this is open to discussion.